addons/interaction/functions/fnc_openDoor.sqf

```diff
@@ -75,6 +78,9 @@ GVAR(usedScrollWheel) = false;
 
             {_house animate [_x, _phase]; false} count _animations;
         };
+
+        // Raise local stopped opening event
+        [QGVAR(doorOpeningStopped)] call CBA_fnc_localEvent;
```

Comment:
Might as well pass the house and door, no?

Reply:
Might as well I suppose

Reply:
> Might as well pass the same arguments as the open EH, no?

The `_door `is not available from the PFH scope, as it's not passed. So either pass only the `_house` and `_animations`, or add the `_door` as a param to the PFH. Which one would be more fitting?

Reply:
Pass `_door`.